src/main/java/racingcar/domain/Cars.java

```diff
@@ -0,0 +1,52 @@
+package racingcar.domain;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.Collectors;
+import racingcar.response.RaceResultResponse;
+
+public class Cars {
+
+    private final int CARS_INPUT_CRITERION = 2;
+    private final List<Car> cars;
+
+    public Cars(Names names) {
+        validateCarsSize(names);
+        this.cars = names.toCarList();
+    }
+
+    public List<Car> getCars() {
+        return Collections.unmodifiableList(cars);
+    }
+
+    public void moveAllCars() {
+        cars.forEach(car -> car.moveForward(RandomNumber.generateRandomNumber()));
+    }
+
+    public int getMaxPosition() {
+        return cars.stream()
+            .mapToInt(Car::getPosition)
+            .max()
+            .orElseThrow(() -> new IllegalStateException("Car 리스트가 존재하지 않습니다"));
+    }
+
+    public List<String> findWinners() {
+        int maxPosition = getMaxPosition();
+        return cars.stream()
+            .filter(car -> car.getPosition() == maxPosition)
+            .map(Car::getName)
+            .collect(Collectors.toList());
```

Comment:
```suggestion
            .toList();
```
Collectors.toList() 보다는 toList 사용이 권장된다고 합니다~